runtime_k8s_scanner/pkg/sbomdb/getter.go

```diff
@@ -55,5 +56,10 @@ func (g *GetterImpl) Get(ctx context.Context, imageHash string) ([]byte, error)
 		}
 	}
 
-	return sbom.Payload.Sbom, nil
+	uncompressedSbom, err := gzip.UnCompress(sbom.Payload.Sbom)
```

Comment:
Maybe we should add a arg to the function whether unzip is needed. We will need a flow where we need to handle it zipped e2e

Reply:
> We will need a flow where we need to handle it zipped e2e

Not sure in what cases, I wanted to make the Get/Set functions unaware of the gzip/unzip logic, also to not break the existing interface.

Reply:
We can add a new GetCompressed method